src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: Roboto, serif;
+
+  @include content-padding-in-line;
```

Comment:
The mixin `content-padding-in-line` is used here, but it is not defined within this file. Ensure that this mixin is defined elsewhere and is correctly imported into this file to avoid compilation errors.